integration/test/plan/README.md

```diff
@@ -11,6 +11,11 @@ Ensure you have completed the creation of your `~/.geopmrc` file
 described in the [integration README](../../README.md#prerequisites).
 
 > [!NOTE]
+> `~/.geopmrc` is only needed if utilizing `run_env.sh` to setup your
```

Comment:
Introduce the prerecs for a tagged release that is installed globally or via a system installed module.  Then explain how to run this test using a development snapshot.